src/main/asciidoc/reference-guide/tutorials/configdatabaseresource.adoc

```diff
@@ -0,0 +1,160 @@
+=== Configure a Database resource
+:hardbreaks:
+
+For Syncope 2.0.x
+
+* Introduction
+* Pre-requisites
+* Configure Database Table connector instance
+* Configure Database Table resource
+* Configure Resource Provision rules
+
+==== Introduction
+
+This tutorial shows all the steps needed to configure an external Database Table resource identified by the following connection parameters:
+
+[cols="2*"]
+|===
+|Host
+|localhost
+
+|Port
+|3306
+
+|User
+|root
+
+|User password
+|password
+
+|JDBC connection URL
+|jdbc:mysql://localhost:3306/demo
+
+|Database
+|demo
+
+|Table
+|test
+|===
+
+==== Pre-requisites
+
+. Download JDBC driver (MySQL JDBC driver in this case) and include it into your JEE container classpath.
+
+. Verify that the bundle exists or https://cwiki.apache.org/confluence/display/SYNCOPE/Install+connector+bundles[Install ConnId Database Table connector bundle]
+
+==== Configure Database Table connector instance
+
+The list of all configurable parameters is available at https://connid.atlassian.net/wiki/display/BASE/Database+Table[Database Table Connector Configuration].
+Use this tutorial as reference for your configuration.
+
+. Open browser and access to the Syncope administration console http://host:port/syncope-console
```

Comment:
It should rather be `protocol://host:port/syncope-console`, according to the rest of this guide